### PR TITLE
Docs 9030 add or link troubleshooting info from kb

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

**Review by fpiacentini08 — DISMISSED** [review]
LGTM

File: modules/ROOT/pages/installing-and-configuring-runtime-manager-agent.adoc [hunk 1]
```diff
@@ -326,9 +326,9 @@ The registration token includes your organization's ID and the current environme
 . Click *Copy* to copy the command.
 . In a terminal window, change to the `$MULE_HOME/bin` directory for the Mule instance that you're registering.
 . Paste the command on the command line.
-. Change the instance name `server-name` to the unique name to use to label Mule in the Runtime Manager console.
 +
-The `server-name` value is unique across environments. The `server-name` can be the same for the same organization in different environments.
+The command includes the unique `server-name` used to label Mule in the Runtime Manager console.
```
Comment (chris-campbell9) [inline modules/ROOT/pages/installing-and-configuring-runtime-manager-agent.adoc]: Asking with a newbie hat on: Does the customer explicitly add the unique server-name when executing the command? Or does the command already include the server-name? Or perhaps this is explained in a previous part of the document?

File: modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc [hunk 2]
```diff
@@ -1,15 +1,84 @@
...
+See the xref:release-notes::runtime-manager-agent/runtime-manager-agent-release-notes.adoc[Runtime Manager Agent Release Notes] and xref:installing-and-configuring-runtime-manager-agent.adoc#update-the-agent[Update the Agent].
+.. Add the Mule server to Runtime Manager.
++
+See xref:servers-create.adoc[Add Servers to Runtime Manager].
+
+
+== Collect Agent Installation Information for Support 
+
+You can use the `$MULE_HOME/bin/amc_debug` script to collect information used by Support to troubleshoot the Anypoint Runtime Manager agent installation issues.
+If you can't solve problems with the Runtime Manager agent, run the script and save the output before filing a support ticket.
```
Comment (chris-campbell9) [inline modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc]: Please consider something like "If you are not successful in resolving the Runtime Manager agent issues...".

"Can't solve problems" may sound a bit critical of the reader's mental capabilities. 😄

File: modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc [hunk 3]
```diff
@@ -1,15 +1,84 @@
...
+. Verify connectivity with all required hosts.
+.. Ensure that your allowlist includes the required hostnames.
++
+See xref:rtm-agent-whitelists.adoc[Ports, IP Addresses, and Hostnames to Whitelist].
+.. Verify that your firewall or proxy server is not intercepting the Runtime Manager certificate.
++ 
+See xref:rtm-agent-proxy-config,adoc#verify-that-the-proxy-server-does-not-modify-the-runtime-manager-certificate[Verify That the Proxy Server Does Not Modify the Runtime Manager Certificate].
+. Verify that the `mule-agent.yml` file is not corrupted or empty.
++
+If the `<MULE_HOME>/conf/mule-agent.yml` file is empty or contains unknown characters:
```
Comment (chris-campbell9) [inline modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc]: Asking with a newbie user hat on: is it explained somewhere how to determine the "MULE_HOME" path? Is the path stored as an environment variable?

How does one know if the agent file is corrupted? Is this a known issue?

I notice that later (line 80) you say $MULE_HOME as opposed to <MULE_HOME>. Not a big deal, but consider changing for consistency.

File: modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc [hunk 4]
```diff
@@ -1,15 +1,84 @@
...
++
+//DASHBOARD SHARED 
+include::partial$cluster-server.adoc[tag=viewServerinClusterOrGroup]
++
+* If the license has expired, see xref:mule-runtime::installing-an-enterprise-license.adoc[Install an Enterprise License].
+* If the certificate has expired, see xref:servers-cert-renewal.adoc[Renew Server Certificates].
+. Verify connectivity with all required hosts.
+.. Ensure that your allowlist includes the required hostnames.
++
+See xref:rtm-agent-whitelists.adoc[Ports, IP Addresses, and Hostnames to Whitelist].
```
Comment (chris-campbell9) [inline modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc:56]: You mention "allowlist" in line 50, but "whitelist" here. Should they both be "allowlist" per our recent inclusion guidelines? 

It looks like the title of the linked page includes "whitelist", so maybe it's beyond the scope of this PR to consider making a change?

File: modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc [hunk 5]
```diff
@@ -1,15 +1,84 @@
...
-You can use the `$MULE_HOME/bin/amc_debug` script to collect information used by Support to troubleshoot the Anypoint Runtime Manager agent installation issues. If you experience issues with the Runtime Manager agent plug-in or with managing applications on Anypoint Platform, run the script and save the output before filing a support ticket.
+Connection issues between your on-premises Mule runtime engine and the Runtime Manager agent might cause discrepancies, such as metrics not appearing in the server monitoring dashboard in Runtime Manager.
+
+== Troubleshoot Connection Issues Between the Agent and Mule
+
+To troubleshoot connection issues:
+
+. Check the JVM you are using.
++
+If you are using the IBM JVM, you must install the truststore manually because this truststore requires different parameters than Oracle JVM or OpenJDK.
```
Comment (chris-campbell9) [inline modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc]: Will "truststore" be familiar to a newbie user? Or does this document assume that a user will be more advanced?

**Comment by chris-campbell9** [comment]
@hannanelson Looks good! Most of my comments are newbie-centric, so they may not apply. (Meaning that the document may be geared towards advanced users).

**Review by chris-campbell9 — DISMISSED** [review]

File: modules/ROOT/pages/installing-and-configuring-runtime-manager-agent.adoc (hunk 1 above)
Comment (hannanelson) [reply modules/ROOT/pages/installing-and-configuring-runtime-manager-agent.adoc]: The customer names the server in RTM and the command includes the server-name the customer-defined. I would have removed this entirely, but I remember adding it in to clarify at the request of someone a while back.

File: modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc (hunk 5 above)
Comment (hannanelson) [reply modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc]: The link gives more info on the truststore.

File: modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc (hunk 4 above)
Comment (hannanelson) [reply modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc:56]: The names haven't changed yet. Still scoping.

File: modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc (hunk 2 above)
Comment (hannanelson) [reply modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc]: Yeah, you're right. I was following Grammarly blindly.

File: modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc (hunk 3 above)
Comment (hannanelson) [reply modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc]: Added some clarification - they will have encountered this when they installed Mule.... over there in the Mule runtime repo.

**Review by chris-campbell9 — APPROVED** [review]

**Pull request merged**